locations/spiders/okq8.py

```diff
@@ -0,0 +1,49 @@
+from scrapy import Spider
+
+from locations.dict_parser import DictParser
+from locations.hours import DAYS, OpeningHours
+
+
+class OKQ8Spider(Spider):
+    name = "okq8"
+    start_urls = [
+        "https://www.okq8.se/-/Station/GetGlobalMapStations?appDataSource=9d780912-2801-4457-9376-16c48d02e688"
+    ]
+
+    BRANDS = {
+        "OKQ8": {"brand": "OKQ8", "brand_wikidata": "Q2789310"},
+        "Tanka": {"brand": "Tanka", "brand_wikidata": "Q10690640"},
+        "IDS": None,  # {"brand": "Ids"} # Some kind of private? HGV?
+        "Eon": None,  # {"brand": "Eon"} # ?
```

Comment:
"Eon" seems to be this https://www.st1biogas.se/hitta-station

Reply:
There was literally 1 Eon and 30 something IDS. I don't think they are big enough to worry about, and I'm throwing them away on line 40 so data consumers don't have to worry either.